README.md

```diff
@@ -71,28 +71,28 @@ export default withUtilityConnect(CreateCredentials, config);
 
 ### Config options
 
-| Name          | Type     | Description                                | Options                              | Required |
-| ------------- | -------- | ------------------------------------------ | ------------------------------------ | -------- |
-| `scope`       | `string` | User flow type                             | `['create', 'update']`               | No       |
-| `env`         | `string` | API environment                            | `['local', 'staging', 'production']` | Yes      |
-| `accessToken` | `string` | API token for authenticating requests      |                                      | Yes      |
-| `client`      | `string` | Name used to reference organization in app |                                      | Yes      |
-| `data`        | `object` | Data passed to the api                     |                                      | Yes      |
-| `callbacks`   | `object` | Callback functions                         |                                      | No       |
-| `uiTheme`     | `string` | UI color theme                             | `['light', 'dark']`                  | No       |
+| Name          | Type     | Description                                | Options                              | Required | Default  |
+| ------------- | -------- | ------------------------------------------ | ------------------------------------ | -------- | -------  |
+| `scope`       | `string` | User flow type                             | `['create', 'update']`               | No       | 'create' |
+| `env`         | `string` | API environment                            | `['local', 'staging', 'production']` | Yes      | none     |
+| `accessToken` | `string` | API token for authenticating requests      |                                      | Yes      | none     |
+| `client`      | `string` | Name used to reference organization in app |                                      | Yes      | none     |
+| `data`        | `object` | Data passed to the api                     |                                      | Yes      | none     |
+| `callbacks`   | `object` | Callback functions                         |                                      | No       | none     |
+| `uiTheme`     | `string` | UI color theme                             | `['light', 'dark']`                  | No       | 'light'  |
 
 #### scope
 
 Specifies the user flow. Defaults to `create`.
 
-- `create`: Opens the Utility Connect service in the "create" flow - input credentials will be used to create a user and corresponding utility credential record.
-- `update`: Opens the Utility Connect service in the "update" flow - input credentials will be used to update an existing utility credential record. Requires that you add `user.id` and `utilityCredential.id` in the `data` object.
+- `create`: Opens the Utility Connect service in the "create user" flow - input credentials will be used to create a user and corresponding utility credential record.
```

Comment:
should this be `create utility credentials` flow? We create both a user and utility credentials but the credentials feel like the primary focus

Reply:
hmmm lets leave for now but I think this is a good question to raise to the group when we try to align the docs